applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -44,20 +103,35 @@ int sw_codec_encode(void *pcm_data, size_t pcm_size, uint8_t **encoded_data, siz
 	case SW_CODEC_LC3: {
 #if (CONFIG_SW_CODEC_LC3)
 		uint16_t encoded_bytes_written;
+		char *lc3_in_data_ptrs[2];
```

Comment:
I went with only `pcm_data_mono_ptr`, as _technically_ this pointer doesn't always point to converted data. Do you think that works, or is it just more confusing since the function is called `sw_codec_sample_rate_covnert()`?